polaris-core/src/main/java/org/apache/polaris/core/monitor/PolarisMetricRegistry.java

```diff
@@ -45,7 +51,11 @@ public class PolarisMetricRegistry {
 
   public PolarisMetricRegistry(MeterRegistry meterRegistry) {
     this.meterRegistry = meterRegistry;
-  }
+    new ClassLoaderMetrics().bindTo(meterRegistry);
+    new JvmMemoryMetrics().bindTo(meterRegistry);
+    new JvmGcMetrics().bindTo(meterRegistry);
+    new ProcessorMetrics().bindTo(meterRegistry);
+    new JvmThreadMetrics().bindTo(meterRegistry);   }
```

Comment:
style

Reply:
yeah, the google formatter is already yelling at me 😅